openedx/core/djangoapps/content/block_structure/models.py

```diff
@@ -255,24 +255,13 @@ def _prune_files(cls, data_usage_key, num_to_keep=None):
         """
         Deletes previous file versions for data_usage_key.
         """
-        if not settings.BLOCK_STRUCTURES_SETTINGS.get('PRUNING_ACTIVE', False):
-            return
-
         if num_to_keep is None:
             num_to_keep = config.num_versions_to_keep()
 
         try:
             all_files_by_date = sorted(cls._get_all_files(data_usage_key))
             files_to_delete = all_files_by_date[:-num_to_keep] if num_to_keep > 0 else all_files_by_date  # lint-amnesty, pylint: disable=invalid-unary-operand-type
             cls._delete_files(files_to_delete)
-            log.info(
```

Comment:
@Yagnesh1998 I'm not sure that we want to delete this log message, although maybe it's worth making it a `DEBUG` message instead.  @ormsbee is this message still useful as an `INFO` message from your perspective?